src/lib/tecs/TECS.cpp

```diff
@@ -412,6 +418,7 @@ void TECSControl::_calcPitchControl(float dt, const Input &input, const Specific
 	const float pitch_increment = dt * param.vert_accel_limit / math::max(input.tas, FLT_EPSILON);
 	_pitch_setpoint = constrain(pitch_setpoint, _pitch_setpoint - pitch_increment,
 				    _pitch_setpoint + pitch_increment);
+	_pitch_setpoint = constrain(_pitch_setpoint, param.pitch_min, param.pitch_max);
```

Comment:
This is not really needed, as pitch_increment above is already constrained.

Reply:
It is if you want that a changed pitch limit has an effect immediately after changing it. Otherwise it has to go through the increment filter (as seen in the crash that happened).